docs/track.rst

```diff
@@ -835,22 +924,210 @@ This is an administrative operation. Metrics are not reported by default. Report
+
+This operation is :ref:`retryable <track_operations>`.
+
+create-component-template
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+With the operation ``create-component-template`` you can execute the `create component template API <https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-component-template.html>`_. It supports two modes: it creates either all component templates that are specified in the track's ``component-templates`` section or it creates one specific component template defined by this operation.
+
+If you want it to create templates that have been declared in the ``component-templates`` section you can specify the following properties:
+
+* ``template`` (optional): If you specify a template name, only the component template with this name will be created.
+* ``settings`` (optional): Allows to specify additional settings that will be merged with the settings specified in the body of the component template in the ``component-templates`` section.
+* ``request-params`` (optional): A structure containing any request parameters that are allowed by the create component template API. Rally will not attempt to serialize the parameters and pass them as is. Always use "true" / "false" strings for boolean parameters (see example below).
+
+If you want it to create one specific template instead, you can specify the following properties:
+
+* ``template`` (mandatory): The name of the template that should be created.
+* ``body`` (mandatory): The body for the create template API call.
+* ``request-params`` (optional): A structure containing any request parameters that are allowed by the create component template API. Rally will not attempt to serialize the parameters and pass them as is. Always use "true" / "false" strings for boolean parameters (see example below).
+
+**Examples**
+
+The following snippet will create all component templates that have been defined in the ``component-templates`` section::
+
+    {
+      "name": "create-all-templates",
+      "operation-type": "create-component-template",
+      "request-params": {
+        "create": "true"
+      }
+    }
+
+With the following snippet we will create a new component template that is not defined in the ``component-templates`` section. Note that we specify the component template settings directly in the body::
+
+    {
+      "name": "create-a-template",
+      "operation-type": "create-composable-template",
+      "template": "component_template_with_2_shards",
+      "body": {
+        "template": {
+          "settings": {
+            "number_of_shards": 2
+          },
+          "mappings": {
+            "_source": {
+              "enabled": false
+            }
+          }
+        }
+      }
+    }
+
+
+.. note::
+    If your component template is used in composable templates ensure these are created after using the ``create-composable-template`` operation.
```

Comment:
Did you mean: "these are created after using the ``create-comonent-template`` operation."?

Reply:
no, but clearer would be `If your component template is used in composable templates, ensure these are created after specifying this operation using the ``create-composable-template`` operation.`